notebooks/checks/performance/regression_error_distribution.ipynb

```diff
@@ -0,0 +1,618 @@
+{
+ "cells": [
```

Comment:
Does the condition pass for 1 and fail for 2?

Reply:
yes